bluesky/tests/test_bec.py

```diff
@@ -80,3 +80,9 @@ def broken_plan(dets):
         yield from bps.trigger_and_read(dets)
 
     RE(broken_plan([hw.det]))
+
+
+def test_live_grid(RE, hw):
+    bec = BestEffortCallback()
+    RE.subscribe(bec)
+    RE(grid_scan([hw.det4], hw.motor1, 0, 1, 1, hw.motor2, 0, 1, 2, True))
```

Comment:
:+1:

Reply:
Thanks for reviewing this PR, @jrmlhermitte! Yes, I agree we may need to specify the extents (a topic for a separate PR).